kotlinx-coroutines-core/common/src/flow/operators/Delay.kt

```diff
@@ -37,39 +37,50 @@ import kotlin.time.*
  *
  * Note that the resulting flow does not emit anything as long as the original flow emits
  * items faster than every [timeoutMillis] milliseconds.
+ * @param timeoutMillis must be positive
  */
 @FlowPreview
 public fun <T> Flow<T>.debounce(timeoutMillis: Long): Flow<T> {
     require(timeoutMillis > 0) { "Debounce timeout should be positive" }
```

Comment:
Why `delay(0)` is good while `debounce(0)` isn't.
Does timeout "should be positive" or "must be positive"?

Please consider:

    if (timeoutMillis <= 0) return this